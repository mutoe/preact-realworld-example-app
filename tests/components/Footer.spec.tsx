import { h } from 'preact'
import render from 'preact-render-to-string'

import Footer from '../../src/components/Footer'

describe('# Footer', () => {
  it('should be display normally', function () {
    const html = render(<Footer />)

    expect(html).toMatchSnapshot()
  })
})
