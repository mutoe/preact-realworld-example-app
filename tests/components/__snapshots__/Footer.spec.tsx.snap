// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Footer should be display normally 1`] = `
<footer>
  <div class="container">
    <a href="/" class="logo-font">conduit</a>
    <span class="attribution">An interactive learning project from
      <a href="https://thinkster.io">Thinkster</a>. Code &amp; design licensed under MIT.</span>
  </div>
</footer>
`;
